src/components/topbar/TopBar.tsx

```diff
@@ -19,7 +19,7 @@ const TopBar: React.FC<TopBarProps> = (props: TopBarProps) => {
 		try {
 			instance.logoutRedirect(request);
 		} catch (e) {
-			console.log(e);
+			alert("There was a problem logging you out. Press F12 for more details.");
```

Comment:
Sorry I was unclear. You shouldn't remove the console.log part, if you do then there will be nothing to see when pressing F12, right? to the whole thing should be:
```suggestion
                             console.log(e);
                             alert("There was a problem logging you out. Press F12 for more details.");
```